<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="mbp.home" name="fr.eseo.atribus.dao.DaoConfigurationExceptionTest" tests="3" failures="0" timestamp="28 May 2016 22:25:36 GMT" time="0.006" errors="0">
  <testcase name="daoConfigurationExceptionString" time="0.003" classname="fr.eseo.atribus.dao.DaoConfigurationExceptionTest"/>
  <testcase name="daoConfigurationExceptionStringThrowable" time="0.001" classname="fr.eseo.atribus.dao.DaoConfigurationExceptionTest"/>
  <testcase name="daoConfigurationExceptionThrowable" time="0.002" classname="fr.eseo.atribus.dao.DaoConfigurationExceptionTest"/>
</testsuite> <!-- fr.eseo.atribus.dao.DaoConfigurationExceptionTest -->
